common/client.go

```diff
@@ -46,19 +46,20 @@ type RestClientConfig struct {
 	Retries       int
 	Credential    *Credential
 	TestMode      bool
+	RootURL       string
 }
 
-func GetDefaultRestClientConfig() RestClientConfig {
-	return RestClientConfig{TimeoutMillis: DefaultRestTimeout, Retries: DefaultRestRetries}
+func GetDefaultRestClientConfig(rootURL string) RestClientConfig {
+	return RestClientConfig{TimeoutMillis: DefaultRestTimeout, Retries: DefaultRestRetries, RootURL: rootURL}
 }
 
 // GetRestClientConfig returns a RestClientConfig based on a ServiceConfig
 func GetRestClientConfig(config ServiceConfig) RestClientConfig {
-	return RestClientConfig{TimeoutMillis: config.Common.Api.RestTimeoutMillis, Retries: config.Common.Api.RestRetries}
+	return RestClientConfig{TimeoutMillis: config.Common.Api.RestTimeoutMillis, Retries: config.Common.Api.RestRetries, RootURL: config.Common.Api.RootServiceUrl}
 }
 
 // NewRestClient creates a new Rest client.
```

Comment:
Change:


// NewRestClient creates a new Romana REST client. It provides convenience
// methods to make REST calls. When configured with a root URL pointing
// to Romana root service, it provides some common functionality useful 
// for Romana services (such as getting configuration from root service, etc.)
// If the root URL does not point to the Romana service, the generic REST operations
// still work, but Romana-specific functionality does not.